routeguide/server/src/main/scala/handlers/RouteGuideServiceHandler.scala

```diff
@@ -44,16 +45,16 @@ class RouteGuideServiceHandler[F[_]: ConcurrentEffect](implicit E: Effect[Task])
       point.findFeatureIn(features)
     }
 
-  override def listFeatures(rectangle: Rectangle): F[Observable[Feature]] = {
-    val left   = Math.min(rectangle.lo.longitude, rectangle.hi.longitude)
-    val right  = Math.max(rectangle.lo.longitude, rectangle.hi.longitude)
-    val top    = Math.max(rectangle.lo.latitude, rectangle.hi.latitude)
-    val bottom = Math.min(rectangle.lo.latitude, rectangle.hi.latitude)
+  override def listFeatures(rectangle: Rectangle): F[Stream[Feature]] = {
```

Comment:
this is where all the compiler errors come from -- the issue is that the sbt-mu-srcgen generates code that uses an fs2 `Stream` for the `listFeatures` method, whereas this was originally implemented using a Monix `Observable` in the `Protocol.scala` hand-written file that was deleted.  My gut here says that I should just refactor all of the handlers to use `Stream`s instead of `Observable`s since that falls in line with the generated code, but I just had two questions: 
1) is there any reason the original example was implemented using `Observable`, and does that behavior need to be retained?
2) is it possible to generate protocol via the sbt plugin that uses `Observabe`s instead of `Stream`s?  The `service.proto` file uses a `stream` type for this method and so I'm not sure if there exists a `observable` type as well.

Reply:
1. This is before my time but I think mu supported `Observable` first and then support for fs2 `Stream` was added later. I guess this is quite an old example. Probably reasonable to keep using `Observable` so we have at least one example of use of `Observable`.

2. Yes, `muSrcGenStreamingImplementation`. See https://higherkindness.io/mu-scala/reference/source-generation#advanced-settings

Reply:
Cool; thanks for sharing your insights!  I'll update the plugin to use `Observable` as the stream type!